### DISCUSSION TITLE
Editor Pane Issue

### DISCUSSION THREAD

**Discussion by maheshrajrp** [eclipse-theia/theia#11155] [General]
Hi,
The Monaco Suggestion are hidden in Theia. The BottomView seems to be hiding those. Whereas in VSCode it's over the BottomView. 
Can this be consider as an issue ?
![image](https://user-images.githubusercontent.com/80001447/168447610-047be51c-0a30-449e-94d0-970e3d082e0d.png)

The same in VSCode.
![image](https://user-images.githubusercontent.com/80001447/168447583-3c0d72b5-1d58-4799-9774-2f2226c30386.png)

**Comment by kittaakos** [comment]
> Monaco Suggestion are hidden in Theia

I have seen this with the _Command Palette_ (<kbd>Ctr/Cmd</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>) too.

**Reply by maheshrajrp** [reply]
![image](https://user-images.githubusercontent.com/80001447/168510912-9e1d3378-34b1-473c-9a34-1a5452f54db6.png)

Yes, Same here too.

**Comment by maheshrajrp** [comment]
Hi @vince-fugnitto ,
Can you provide any suggestions ?

**Reply by kittaakos** [reply]
The _Command Palette_ seems to work with 1.25.0. The content assist still has the overlay issue:

```
Version 1.25.0 (Beta)
VS Code API Version: 1.53.2
```

Content assist:
<img width="552" alt="Screen Shot 2022-05-18 at 10 27 43" src="https://user-images.githubusercontent.com/1405703/168994158-0aa9c28c-36e4-4d14-b50d-5714394bf2db.png">

Command palette:
<img width="706" alt="Screen Shot 2022-05-18 at 10 27 49" src="https://user-images.githubusercontent.com/1405703/168994165-1bc04f3c-0cf0-49d2-8470-fecb3958bea7.png">

**Reply by vince-fugnitto** [reply]
> Hi @vince-fugnitto ,Can you provide any suggestions ?

I don't really have any suggestions per se rather that fix the bug and ensure that monaco has priority, we don't really play that well with monaco hovers or overlays, we had issues in the past as well:
- https://github.com/eclipse-theia/theia/issues/2417
- https://github.com/eclipse-theia/theia/issues/6272
